signer/signer.go

```diff
@@ -79,8 +79,9 @@ type PolicyInformation struct {
 
 // ProfileConfig describes the certificate issuance constraints
 type ProfileConfig struct {
-	AllowRSAKeys    bool
-	AllowECDSAKeys  bool
+	UseForRSAKeys   bool
```

Comment:
This naming is somewhat confusing, `UseForRSALeaves` might be clearer.

Reply:
Good call, done.